layouts/shortcodes/covid.html

```diff
@@ -1,12 +1,9 @@
-<div class="alert alert-warning alert-dismissible fade show" role="alert">
-  <strong>Viktig om Covid-19 (koronavirus)</strong>
-  <br>
-  Besøk er ikke tillatt hvis du har symptomer på koronavirus. Hvis du er frisk,
-  kan jeg tilby desinfeksjonsmiddel under besøket. Les mer om symptomer, karantene og isolasjon på
-  <a href="https://www.fhi.no/sv/smittsomme-sykdommer/corona/" class="alert-link">
-    Folkehelseinstituttets nettsider
-  </a>.
-  <button type="button" class="close" data-dismiss="alert" aria-label="Close">
+<!-- Brukes på miraclelanding -->
+<!-- Brukes på visitt-sidene -->
+
+<div class="alert alert-primary alert-dismissible fade show" role="alert">
+  Har du luftveisinfeksjoner eller andre symptomer på covid-19? Da skal du ikke komme på besøk til meg. Les mer om <a href="https://www.fhi.no/nettpub/coronavirus/fakta/hvis_du_mistenker_at_du_er_syk_med_covid-19/" target=_blank class="alert-link">hva du skal gjøre hvis du mistenker at du er smittet på Folkehelseinstituttets sider</a>. Hvis du er frisk, er du velkommen uten håndhilsing, med en meters avstand og hyppig håndvask.
```

Comment:
```suggestion
  Har du luftveisinfeksjoner eller andre symptomer på covid-19? Da skal du ikke komme på besøk til meg. Les mer om <a href="https://www.fhi.no/nettpub/coronavirus/fakta/hvis_du_mistenker_at_du_er_syk_med_covid-19/" target="_blank" class="alert-link">hva du skal gjøre hvis du mistenker at du er smittet på Folkehelseinstituttets sider</a>. Hvis du er frisk, er du velkommen uten håndhilsing, med en meters avstand og hyppig håndvask.
```

HTML-attributter skal alltid anføres i anførselstegn

Reply:
Beklager.